### ISSUE TITLE
Having issues in running docker

### ISSUE THREAD

**Issue by AyeshaNirma** [Ir1d/DARKFACE_eval_tools#7]
Hi Guys,

There are some issue in when I try to run docker. Specifics are
This is the command I run:

sudo docker run --rm -it -v /home/ayesha/user_output_path2/:/tools/data -v /home/ayesha/:/tools/output scaffrey/eval_tools myTester ./data/gt/ /tools/data/

1) It reads all prediction files(in folder user_output_path2) and in the end display correct AP for DSFD, PyramidBox etc. However, for my algorithm (myTester) it gives AP equal to NAN. Now I am using your sample results file as predictions (ones provided by UG2 challenge) I have plotted them on the 100 images and they seem reasonable. So Nan AP is basically some technical issue. 

2) It also says when I run the above command "Can not find the gt file ./data/gt/54.txt"

3) Why does your code also says "Norming prediction", when the predictions are already normalized in sample txt files between 0-1 ?

Thanks anyways

Thanks anyways

**Comment by Ir1d** [comment]
Hi,
1. (and 2.) I think it's because the ground truth bounding boxes are missing, so the command fails. Did the dry-run instruction include the ground truth files for track2.2 in UG2 challenge? (or only sample prediction results are provided)
3. it norms the confidence score into [0,1], then we enumerate the threshold from 0 to 1 by 1/1000, to plot the precision-recall curve.

**Comment by AyeshaNirma** [comment]
At the moment only predictions are shared for 2.2. I thought "gt" is already in the docker and locked. I guess I was wrong.
 
So basically as of now we cannot evaluate any thing on those 100 images released ?

Finally, the DSFD results(0.4804) are these on 100 images or on the whole testing set of DarkFace ?  and more importantly did you guys fine-tune DSFD on DarkFace ? 

I am assuming you are related to the Arxiv report either as an author or collaborator.


Thanks anyways

**Comment by Ir1d** [comment]
Sorry for the inconvenience, the following file should contain the ground truth labels. The docs will be updated soon.

```
Link:  https://pan.baidu.com/s/13kkQf439ca5UwSMgYROsZw
Extracting code: w304
Link (Google Drive):
https://drive.google.com/open?id=1BcgmJ292r1D_umbxZotzGY5PlBPqqQgc
The path after uncompressing: user_label_path2
```
`0.4804` is on the whole testing set of DarkFace(which is used in the challenge). This result is obtained using DSFD models pre-trained on WIDER FACE.
